api/src/api/notes.js

```diff
@@ -45,6 +45,29 @@ router.get(
   }),
 );
 
+//GET /notes/labels
+router.get(
+  '/labels',
+  requireRegistered,
+  errorWrap(async (req, res) => {
+    const notes = await Note.find({});
+    let labelList = [];
+
+    for (const note of notes) {
+      for (const label of note.metaData.labels) {
+        if (!labelList.includes(label)) {
+          labelList.push(label);
+        }
+      }
+    }
+
+    res.status(200).json({
```

Comment:
Add a `message` response field for the frontend. Not super important but good practice nonetheless. Something like "Notes retrieved successfully."